.dockerignore

```diff
@@ -4,7 +4,5 @@
 # now include specific items as our inclusions < our exclusions
 !connect
 !local-certs
-!setup.*
```

Comment:
removing the setup.py removed the dependency on the README and LICENSE. Although we may need to include the license in a container. We can check into that.